API.md

```diff
@@ -603,6 +604,13 @@ db.events.on('write', (dbname, hash, entry) => ... )
 
 Emitted after an entry was added locally to the database. *hash* is the IPFS hash of the latest state of the database. *entry* is the added database op.
 
+### `peer`
+```javascript
+db.events.on('peer', (peer) => ... )
+```
+
+Emitted when a new peer connects via ipfs pubsub. *peer* is the new peer,  an instance of [`peerInfo`](https://github.com/libp2p/js-peer-info)
```

Comment:
Is `peer` an instance of `peerInfo` or is it just the peer-id string?

Reply:
I didn't check back far enough into the js-ipfs implementation. Seems like its just the peer-id string.